docs/operators_and_observables.ipynb

```diff
@@ -0,0 +1,953 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "WZ1G8QHhdHZR"
+   },
+   "source": [
+    "##### Copyright 2020 The Cirq Developers"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "KQa9t_gadIuR"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
```

Comment:
Remove this `@title`

Reply:
We need that for the site!

Reply:
Seems like I've been super out of the loop when it comes to docs lately.